__tests__/identity.test.js

```diff
@@ -1,64 +1,70 @@
-// const { v4: uuidv4 } = require('uuid')
-// const { apiUrl, clientRegistrationToken, idRealmName, idAppName } = global
-// const Tozny = require('../node')
-// const ops = require('./utils/operations')
+const { v4: uuidv4 } = require('uuid')
+const { apiUrl, idRealmName, idAppName, clientRegistrationToken } = global
```

Comment:
I think this token needs to get generated with the ability to add 'identity' client types, and I'm not sure off the top of my head if 'normal' tokens in the dashboard allow it... Any token created during realm creation would work for both identity and general clients, though.